keyboards/hub16/keymaps/ahk_companion/keymap.c

```diff
@@ -0,0 +1,157 @@
+#include QMK_KEYBOARD_H
+
+#if defined(RGBLIGHT_ENABLE) && defined(RGBLIGHT_LAYERS)
+static uint32_t rgb_preview_timer = 0;
+#endif
+extern rgblight_config_t rgblight_config;
```

Comment:
```suggestion
```

Reply:
Is the request to remove that line? Just wondering why...Although I don't fully understand it's need (spidey3 helped me with it) but it was for something RGB related.

Reply:
Yup. 

And it's because most of these settings are exposed in functions.

https://docs.qmk.fm/#/feature_rgblight?id=query